test/kubetest/k8s.go

```diff
@@ -406,6 +407,10 @@ func NewK8s(g *WithT, prepare bool) (*K8s, error) {
 		defer wg.Done()
 		client.cleanupFunc = InitSpireSecurity(client)
 	}()
+	go func() {
+		defer wg.Done()
+		client.CreatePod(pods.PrefixServicePod(client.namespace))
```

Comment:
I guess you have to update `prepare` section

Reply:
- [ ] add cleanup of "prefix-service" pod in prepare section